gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -72,18 +78,14 @@ def show_df(df: pd.DataFrame, show: bool) -> None:
 
 
 def plot_overall_return(
-    df: pd.DataFrame, df_m: pd.DataFrame, n: int, m_tick: str, plot: bool = False
+    comb: pd.DataFrame, m_tick: str, plot: bool = False
 ) -> ImageReader:
     """Generates overall return graph
 
     Parameters
     ----------
-    df : pd.DataFrame
-        The dataframe to be analyzed
-    df_m : pd.DataFrame
-        The dataframe for historical market performance
-    n : int
-        The number of days to include in chart
+    comb : pd.DataFrame
```

Comment:
Just a small comment that doesn't need to be addressed now:  comb is a very non-descriptive variable name.  If it is representing returns, I would expect the df to be named something more related to that.